modules/040-node-manager/docs/FAQ.md

```diff
@@ -1250,3 +1250,51 @@ Done
 ```
 
 {% endraw %}
+
+### How to use NodeGroup with priority?
+
+You can use the `priority` field in the NodeGroup CRD to specify how to order the cluster nodes.
+For example, you can try to order the spot-nodes first, and if they run out, the regular nodes. Or you could try to
+to order larger nodes first, and if the cloud resources are exhausted, to order smaller nodes.
+
+An example using a spot-node:
+
+Create 2 ng:
+
+```yaml
+---
+apiVersion: deckhouse.io/v1
+kind: NodeGroup
+metadata:
+  name: worker-spot
+spec:
+  cloudInstances:
+    classReference:
+      kind: AWSInstanceClass
+      name: worker-spot
+    maxPerZone: 5
+    minPerZone: 0
+    priority: 50
+  nodeType: CloudEphemeral
+---
+apiVersion: deckhouse.io/v1
+kind: NodeGroup
+metadata:
+  name: worker
+spec:
+  cloudInstances:
+    classReference:
+      kind: AWSInstanceClass
+      name: worker
+    maxPerZone: 5
+    minPerZone: 0
+    priority: 30
+  nodeType: CloudEphemeral
+```
+
+In this case, cluster-autoscaler will try to order a spot-node. If it fails to add a node to the cluster within 15 minutes, then
+ng `worker-spot` will be paused (for 20 minutes) and cluster-autoscaler will start ordering nodes from ng `worker`.
+If another node is needed after 30 minutes for the cluster load, it will again try to order from ng `worker-spot` first and only then from ng `worker`.
```

Comment:
```suggestion
The [priority](cr.html#nodegroup-v1-spec-cloudinstances-priority) field of the `NodeGroup' CustomResource allows you to define the order in which nodes will be provisioned in the cluster. For example, `cluster-autoscaler` can first provision *spot-nodes* and switch to regular ones when they run out. Or it can provision larger nodes when there are plenty of resources in the cluster and then switch to smaller nodes once cluster resources run out.

Here is an example of creating two `NodeGroups` using spot-node nodes:
```